content/en/serverless/step_functions/troubleshooting.md

```diff
@@ -4,44 +4,33 @@ title: Troubleshooting Serverless Monitoring for AWS Step Functions
 
 ## I cannot see any traces
 
-### Verify that your Step Function is configured to send all logs
+#### Verify that your Step Function is configured to send all logs
 - Ensure that the `DD_TRACE_ENABLED` tag is set to `true` on the Step Function in your AWS console.
 - In your AWS console, open your Step Function's logging tab. Ensure that _Log level_ is set to `ALL`, and that _Include execution data_ is selected.
 - Ensure that the CloudWatch log group (also found on the logging tab) has a subscription filter to the Datadog Lambda Forwarder in the same region.
 
-### Verify that logs are forwarded successfully to Datadog
+#### Verify that logs are forwarded successfully to Datadog
 - Check the Datadog Lambda Forwarder for error messages. Ensure that you have correctly set your API key and Datadog site.
 - Enable `DEBUG` logs on the Datadog Lambda Forwarder by setting the environment variable `DD_LOG_LEVEL` to `debug`.
 
-### Verify that logs are searchable on Live Search and have DD_TRACE_ENABLED tag
+#### Verify that logs are searchable on Live Search and have DD_TRACE_ENABLED tag
```

Comment:
Bumped these down to H4 because the sidebar feels to cluttered with all these sub-headings

<img width="1205" alt="Screenshot 2025-03-31 at 4 37 47 PM" src="https://github.com/user-attachments/assets/47c9f0a2-0126-4d77-9d70-6269ffd8bb85" />